database/delete.go

```diff
@@ -0,0 +1,27 @@
+package database
+
+import (
+	"database/sql"
+	"errors"
+
+	_ "modernc.org/sqlite"
+)
+
+func Delete(id string) error {
+	var err error
+
+	// Открываем базу данных
+	base, err = sql.Open("sqlite", Path)
+	if err != nil {
+		return errors.New(`Ошибка при инициализации подключения к базе данных: ` + err.Error())
```

Comment:
Ошибки нужно оборачивать
```suggestion
		return fmt.Errof("ошибка при инициализации подключения к базе данных: %w" + err)
```

Reply:
В описании к fmt.Errof :
> If the format specifier includes a %w verb with an error operand, the returned error will implement an Unwrap method returning the operand
Получается тогда можно будет обратно ошибку развернуть (Unwrap)

Reply:
Мне по всему проекту все ошибки переделать?

Reply:
да можно не переделывать, главное понять принцип